docs/modules/dev-services/pages/index.adoc

```diff
@@ -42,6 +42,7 @@ Your integration tests will also automatically use the Arconia Dev Services with
 
 * xref:lgtm.adoc[Grafana LGTM]
```

Comment:
Same thing here. And keep the alphabetical sorting within the _Databases_ group.

* Databases
** xref:mariadb.adoc[MariaDB]
** xref:mysql.adoc[MySQL]
** xref:postgresql.adoc[PostgreSQL]
** xref:redis.adoc[Redis]
* xref:lgtm.adoc[Grafana LGTM]
* xref:ollama.adoc[Ollama]

Reply:
Done